pinot-filesystem/src/test/java/com/linkedin/pinot/filesystem/LocalPinotFSTest.java

```diff
@@ -147,13 +160,5 @@ public void testFS() throws Exception {
     Assert.assertTrue(localPinotFS.exists(secondTestFileUri));
     localPinotFS.copyToLocalFile(testFile.toURI(), new File(secondTestFileUri));
     Assert.assertTrue(localPinotFS.exists(secondTestFileUri));
-
-    // List files on a file - exception if file already exists
-    try {
-      localPinotFS.listFiles(thirdTestFile.toURI());
-      fail();
-    } catch (IllegalArgumentException e) {
-
-    }
   }
 }
```

Comment:
Miss new line at the end of the line.

Reply:
Hmm, I think I removed this part of the code